### PR TITLE
Make all the UI

### LINKED ISSUE #15: Activities admin panel
Currently, the admin panel is split into "Activities Overview" and "Activities Pages".

If I understood correctly, an "activity overview" is the little activity box on the homepage and an "activity page" is the detailed page for an activity.

For me it seems confusing to have to create twice the information about an activity (once for the overview, once for the page), I asked @auloin about it in a previous PR : https://github.com/openknowledgebe/website/pull/10#discussion_r528170162

I would like to open the discussion here.
Do we have activities that won't have a dedicated page ?

### LINKED ISSUE #17: CMS: Story slug and url
At the moment, the slug for a story is `launch-of-civic-lab-brussels-20201201174631.md` and the url is `/stories/launch-of-civic-lab-brussels-20201201174631`.

The datetime in both slug and url is the datetime when the story is created. It should be only the **date** of the story itself (and not the moment the story is created).

The slug format should be `20170505-launch-of-civic-lab-brussels.md` and the url should be `/stories/2017/05/05/launch-of-civic-lab-brussels/` (so we keep the same format as the current posts).

### PR DESCRIPTION
**Pull request by auloin**
Close #15 
Close #17

### CODE REVIEW

**Comment by jbelien** [comment]
I'll check it this weekend 👍 
Could you make sure the build is working before I check ?

File: src/cms/previews/Activity.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+const Activity = ({ entry }) => {
+  const { data } = entry.toJS();
+
+  return (
+    <StyleInjector>
+      <div style={{ marginBottom: '10px' }}>
+        <small>Hover the box bellow to see the tags.</small>
+      </div>
+      <ActivityCard {...data} />
```
Comment (jbelien) [inline src/cms/previews/Activity.jsx:14]: Activity overview logo doesn't appear in the preview.

File: src/components/UI/Person.jsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  & .socials > * {
+    margin-left: 1rem;
+  }
+`;
+
+const Person = ({ name, picture, socials, task }) => {
+  return (
+    <StyledPerson>
+      {picture ? (
+        <Img alt={name} src={picture} />
```
Comment (jbelien) [inline src/components/UI/Person.jsx:34]: Picture upload doesn't seem to be available in the admin.

File: src/pages/index.js [hunk 3]
```diff
@@ -44,22 +44,27 @@ const pinnedActivities = [
 ];
```
Comment (jbelien) [inline src/pages/index.js:45]: Could you replace this hardcoded array by "real" activities from the CMS ?
I didn't find the way to pin an activity in the admin panel.

File: src/pages/team.js [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+export const TeamTemplate = ({ data }) => {
+  return (
+    <>
+      <Header>
+        <Title as="h1">
+          Our <br />
+          Team
+        </Title>
+        <Img src={data?.header?.featured_image} alt="" />
```
Comment (jbelien) [inline src/pages/team.js:103]: Display a box with a black border in the admin preview ... Should be replaced by a placeholder!

**Review by jbelien — CHANGES_REQUESTED** [review]
- [ ] Could you add some "random" content **everywhere** ? For now it's a bit difficult to have an idea what the pages look like and some functionalities in the admin panel doesn't seem to work.
- [ ] Could you disable Cloudinary support during development ?
- [ ] When I try to create a new "Activity Page", the name is a listbox but it's empty (even when I already created an "Activity Overview".

File: src/pages/index.js (hunk 3 above)
Comment (auloin) [reply src/pages/index.js:45]: I didn't connect the CMS to the homepage, my next task is to connect everything together.
To pin an activity go to `Pages > Home > Activities Section > Featured Activities` , same goes for a story.

File: src/components/UI/Person.jsx (hunk 2 above)
Comment (auloin) [reply src/components/UI/Person.jsx:34]: Correct, this is to be added. I'll have to review all other `image` fields so I plan to add it while doing that.

File: src/cms/previews/Activity.jsx (hunk 1 above)
Comment (auloin) [reply src/cms/previews/Activity.jsx:14]: The logo first needs to be uploaded on Cloudinary. The currently in used cloud is one I setup myself, I can switch to okbe one anytime.

File: src/pages/team.js (hunk 4 above)
Comment (auloin) [reply src/pages/team.js:103]: Is this linked to the request to add random content everywhere? Should I make random previews for everything?

File: src/cms/previews/Activity.jsx (hunk 1 above)
Comment (jbelien) [reply src/cms/previews/Activity.jsx:14]: Okay, could you completely disable Cloudinary for now then ?

File: src/pages/team.js (hunk 4 above)
Comment (jbelien) [reply src/pages/team.js:103]: No, it's not linked.
For all the previews, I would add a placeholder (like you did for the Person preview).

File: static/admin/config.yml [hunk 5]
```diff
@@ -107,12 +107,15 @@ collections:
     label_singular: 'Story'
     folder: content/stories
     create: true
-    slug: '{{title}}-{{year}}{{month}}{{day}}{{hour}}{{minute}}{{second}}'
+    slug: '{{year}}{{month}}{{day}}-{{title}}'
     fields:
       - label: 'Date'
         name: 'date'
-        widget: 'date'
+        widget: 'datetime'
```
Comment (jbelien) [inline static/admin/config.yml:114]: Since we only need/use the date, why do you go back to the datetime widget (instead of date widget) ?

Comment (auloin) [reply static/admin/config.yml:114]: Date is deprecated: [date widget](https://www.netlifycms.org/docs/widgets/#date)

Comment (jbelien) [reply static/admin/config.yml:114]: That's indeed ... an EXCELLENT reason.
Sorry :P

**Pull request merged**